Casks/v/virtualdj.rb

```diff
@@ -0,0 +1,31 @@
+cask "virtualdj" do
+  version "2024.b8124"
+  sha256 "fddcdd538b4b1a8a70b7cba07f1f1303c73a387eeae3e7600eccdf96c9ec2e08"
+
+  url "https://download.virtualdj.com/install_virtualdj_#{version.dots_to_underscores}_mac.pkg"
+  name "VirtualDJ"
+  desc "#1 Most Popular DJ Software"
+  homepage "https://virtualdj.com/"
+
+  livecheck do
+    url "https://virtualdj.com/download/mac"
+    regex(/install_virtualdj_?(\d+_b\d+)_mac\.pkg/i)
+    strategy :header_match do |headers, regex|
+      headers["location"].scan(regex).map do |match|
+        match[0].tr("_", ".").to_s
+      end
+    end
+  end
+
+  pkg "install_virtualdj_#{version.dots_to_underscores}_mac.pkg"
```

Comment:
```suggestion
  pkg "install_virtualdj_#{version.csv.first}_b#{version.csv.second}_mac.pkg"
```

Update to match new `version` format.